tfhe/src/shortint/engine/server_side/div_mod.rs

```diff
@@ -97,7 +97,12 @@ impl ShortintEngine {
         scalar: u8,
     ) -> EngineResult<()> {
         assert_ne!(scalar, 0, "attempt to divide by zero");
-        let lookup_table = self.generate_lookup_table(server_key, |x| x / (scalar as u64))?;
+
+        let lookup_table = self.generate_msg_lookup_table(
```

Comment:
not sure I like this style of lookup generation, it hides the parts where there are moduli especially for "low level" usage as we do here 🤔 

is there still a raw primitive without automated modulus management ?

Reply:
Sure, `generate_lookup_table` still exists and is used in many places, especially in the integer API

Reply:
ok, I'll check more thorougly but the changes seemed straightforward otherwise